bundles/AdminBundle/Resources/public/js/pimcore/object/tags/link.js

```diff
@@ -90,6 +90,10 @@ pimcore.object.tags.link = Class.create(pimcore.object.tags.abstract, {
             text = this.data.path;
         }
 
+        if (text.length > 100) {
+            text = text.slice(0, 97)
+            text += '...'
```

Comment:
```suggestion
            text += '…'
```
Let's use ellipsis here 😉

Reply:
👍 … and in that case it is also possible to use `text.slice(0, 99)`